gendo/bot.py

```diff
@@ -31,10 +31,12 @@ def config_from_yaml(cls, path_to_yaml):
             settings = yaml.load(ymlfile)
             return cls(settings=settings)
 
-    def listen_for(self, rule, **options):
+    def listen_for(self, rule):
         def decorator(f):
-            self.add_listener(rule, f, **options)
-            return f
+            def wrapped(**kwargs):
+                self.add_listener(rule, f, **kwargs)
+                return f
+            wrapped()
```

Comment:
The original code looks right to me; do you have a test case showing it fails?

Since you're calling wrapped here and not returning it, `kwargs` will equal `{}` always, won't it?